EnglishFilter/sections/specific.txt

```diff
@@ -256,6 +256,7 @@ feetporno.com,familyporn.tv,freehardcore.com,webvoyeur.com,moviesand.com,momvids
 !
 !
 ||hulkstream.com/jvpop*.js
+imagetwist.com##div[style^="width:300px; height:250px;display: inline-block;"]
```

Comment:
Also add this `imagetwist.com#?#.text-center > iframe[name^="spot_id_"]:upward(1)` to the `extended_css.txt`.

Reply:
Ok. I am not good in fixing ads, but nevermind.

Reply:
@zloyden How to use write extension CSS rules?

Reply:
When you see that using basic rules is not enough.
About writing ext-css rules you can read the information [here](https://kb.adguard.com/en/general/how-to-create-your-own-ad-filters#extended-css-selectors).